implementation/pom.xml

```diff
@@ -49,7 +49,10 @@
     <dependency>
       <groupId>io.smallrye</groupId>
       <artifactId>smallrye-config</artifactId>
-      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.smallrye.config</groupId>
+      <artifactId>smallrye-config</artifactId>
```

Comment:
Not sure if order matters? Should the new dependency be above the older relocated one?

Reply:
It doesn't matter.  It's purely a stylistic choice.

Reply:
So the newer one will always get chosen because the relocate points to io.smallrye.config:smallrye-config:1.5.0, and updated versions would have a higher version than that?

Reply:
Right.  That's the theory anyway. :-)

Reply:
Gotcha